### PR TITLE
Add python-neo4j-driver for aclpwn from AUR

### PR DESCRIPTION
**Pull request by aancw**

### CODE REVIEW

File: packages/python-neo4j-driver/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+pkgver=1.7.6
+pkgrel=1
+pkgdesc='Neo4j Bolt driver for Python'
+arch=('any')
+url='https://github.com/neo4j/neo4j-python-driver'
+license=('Apache')
+depends=('python' 'python-pytz')
+makedepends=('python-setuptools')
+source=("https://github.com/neo4j/$_pkgname/archive/$pkgver.zip")
+sha256sums=('00752aef0b36b54ec5badbebda68d8a0c9df3f8101aa2a49f5dbf975c59d1b4b')
```
Comment (Edu4rdSHL) [inline packages/python-neo4j-driver/PKGBUILD:18]: Use `sha512sums`.

File: packages/python-neo4j-driver/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+package() {
+  cd "$srcdir/$_pkgname-$pkgver"
```
Comment (Edu4rdSHL) [inline packages/python-neo4j-driver/PKGBUILD:21]: Use `cd $_pkgname-$pkgver`, there isn't need to call `$srcdir` or use quotes.

Also add newline between `cd *` line and `python *` line. Please see our [PKGBUILD templates](https://github.com/BlackArch/blackarch-pkgbuilds).

File: packages/python-neo4j-driver/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  python setup.py install --root="$pkgdir/" --optimize=1
+}
```
Comment (Edu4rdSHL) [inline packages/python-neo4j-driver/PKGBUILD:23]: Add final newline.

File: packages/python-neo4j-driver/PKGBUILD (hunk 2 above)
Comment (aancw) [reply packages/python-neo4j-driver/PKGBUILD:21]: I'm following PKGBUILD-python-lib template for `cd srcdir` https://github.com/BlackArch/blackarch-pkgbuilds/tree/master/PKGBUILD-python-lib

Comment (Edu4rdSHL) [reply packages/python-neo4j-driver/PKGBUILD:21]: Looks like it's _wrong_ there! But please follow my changes request. Also, please add a `build()` step, I forget that.

File: packages/python-neo4j-driver/PKGBUILD (hunk 3 above)
Comment (Edu4rdSHL) [reply packages/python-neo4j-driver/PKGBUILD:23]: As you add `build()` step, add `--skip-build` when packaging.

File: packages/python-neo4j-driver/PKGBUILD (hunk 2 above)
Comment (aancw) [reply packages/python-neo4j-driver/PKGBUILD:21]: Okay. Will do.

File: packages/python-neo4j-driver/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+build(){
+  cd $_pkgname-$pkgver
+
+  python setup.py build
+}
+
+package() {
+  cd $_pkgname-$pkgver
+
+  python setup.py install --root="$pkgdir/" --prefix=/usr -01 --skip-build
```
Comment (Edu4rdSHL) [inline packages/python-neo4j-driver/PKGBUILD:29]: Please fix this typo. It's `O1` (optimize) not `01`. Every time you make a change, test with ba-dev.

Comment (aancw) [reply packages/python-neo4j-driver/PKGBUILD:29]: Oh sorry, that was typo. I'm running ba-dev first before submitting to repo.

**Comment by Edu4rdSHL** [comment]
Thank you a lot man! Merging now. As reminder, do not forge to add new/modified/bumped/* packages to `lists/to-release`.

>

**Comment by aancw** [comment]
Ah, thanks for the information and reminder. I don't know the procedure. Will do in later packaging. Thanks!

**Comment by Edu4rdSHL** [comment]
Don't worry mate, we are here to help and learn.

**Pull request merged**